bmwqemu.pm

```diff
@@ -215,11 +215,17 @@ sub log_call {
     else {
         # key/value pairs
         my @result;
+        my $should_stash = 0;
+        my @tmp_array = @_;
+        while (my ($key, $value) = splice(@tmp_array, 0, 2)) {
```

Comment:
I am not sure why you need extra loop for this , why simply not do same things in loop which we already have which started in line 223 ?